server/package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
I would **really prefer** if you do not include 7,404 changed lines to package-lock.json when only 2 lines are changed in package.json. I strongly recommend restoring this file to it's original condition from the main branch, then running `npm install` with just the 2 lines changed in package.json. I suspect that nowhere near 7 thousand lines will change after doing this.

Please respond to this comment before moving forward!

Reply:
I've created a test PR branched off of this PR, where I've made the above requested changes to package-lock.json. Only 1,000 lines are changed there. I've done this by deleting package-lock.json, and running `npm install`. See https://github.com/bravetechnologycoop/BraveSensor/pull/256 . I'd appreciate it if you did the same for this PR and updated package-lock.json to be the new file!

Reply:
fixed!!